frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -176,46 +176,43 @@ local footerTextGeneratorMap = {
         local symbol_type = footer.settings.item_prefix
         local prefix = symbol_prefix[symbol_type].battery
         local powerd = Device:getPowerDevice()
-        local batt_lvl = powerd:getCapacity()
+        local batt_lvl = 0
+        local is_charging = false
+
+        if Device:hasBattery() then
+            local main_batt_lvl = powerd:getCapacity()
+
+            if Device:hasAuxBattery() and powerd:isAuxBatteryConnected() then
+                local aux_batt_lvl = powerd:getAuxCapacity()
+                is_charging = powerd:isAuxCharging()
+                -- Average both batteries to compute the icon...
+                if symbol_type == "icons" or symbol_type == "compact_items" then
+                    prefix = powerd:getBatterySymbol(is_charging, math.floor((main_batt_lvl + aux_batt_lvl) / 2))
+                end
+                -- But sum 'em for the actual text
+                batt_lvl = main_batt_lvl + aux_batt_lvl
```

Comment:
Maybe you want to straighten the logic:
```suggestion
                -- Sum 'em for the actual text
                batt_lvl = main_batt_lvl + aux_batt_lvl
                -- But average both batteries to compute the icon...
                if symbol_type == "icons" or symbol_type == "compact_items" then
                    prefix = powerd:getBatterySymbol(is_charging, math.floor(batt_lvl / 2))
                end
```
Maybe also move `batt_lvl` upwards in the next else branch (for consitancy)?